modules/ROOT/pages/salesforce/salesforce-connector-97.adoc

```diff
@@ -0,0 +1,1274 @@
+* Remote Call-In - An external system accesses, changes, deletes or adds data in Salesforce, and vice versa (Salesforce to external system).
+* Fire and Forget Remote Process Invocation - Salesforce initiates a process in a third-party system and receives an acknowledgment that the process has started. The third-party system continues processing independent of Salesforce.
+* Request-Reply Remote Process Invocation - Salesforce kicks off a process in a remote system, waits for the remote system to finish processing, then accepts control back again from the remote system.
+* User Interface Update Based on Data Changes - The Salesforce UI updates in response to a change in a third-party system.
+
+The Salesforce connector works with the following APIs.
+
+=== Apex SOAP API
+
+The Salesforce https://www.salesforce.com/us/developer/docs/apexcode/[Apex SOAP API] exposes Apex class methods as custom SOAP Web service calls. This allows an external application to invoke an Apex Web service to perform an action in Salesforce.
+
+=== SOAP API
+
+The Salesforce http://www.salesforce.com/us/developer/docs/api/index.htm[SOAP API] provides secure access to your organization's information on Salesforce. Most of the operations that the Salesforce connector performs map to this API.
+
+All the Salesforce operations performed through the SOAP API have an optional parameter called "Headers" that can take any of the following https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/soap_headers.htm[Salesforce SOAP Headers]:
+AllOrNoneHeader, AllowFieldTruncationHeader, AssignmentRuleHeader, CallOptions,
+EmailHeader, LocaleOptions, MruHeader, OwnerChangeOptions, QueryOptions,
+UserTerritoryDeleteHeader, DuplicateRuleHeader
+
+=== Apex REST API
+
+The Salesforce https://developer.salesforce.com/page/Creating_REST_APIs_using_Apex_REST[Apex REST API] creates your own REST-based web services using Apex. It has all of the advantages of the REST architecture, and provides the ability to define custom logic and includes automatic argument/object mapping.
+
+=== Bulk API
+
+The Salesforce https://www.salesforce.com/us/developer/docs/api_asynch/[Bulk API] quickly and securely load batches of your organization's data into Salesforce. See also: <<Load Data in Batches>>.
+
+=== Streaming API
+
+The Salesforce http://www.salesforce.com/us/developer/docs/api_streaming/[Streaming API] securely receives notifications for changes to your organization's information in Salesforce. See <<Receive Inbound Data From Salesforce>> for more information on the use of the Streaming API.
+
+=== Metadata API
+
+The Salesforce http://www.salesforce.com/us/developer/docs/api_meta/[Metadata API] manages customizations and build tools that work with the metadata model, not the data itself.
+
+== Configure in Design Center
+
+. In Anypoint Platform > *Design Center*, click *Create* > *Mule Application*.

```

Comment:
Not sure if this way of referring to selections is correct. Also the button is **Create** and then the next button to click is **Create new application**.

Maybe say:

From the Anypoint Platform Design Center, click **Create**, and then click **Create new application**.

Reply:
Fixed as requested.